From 866c0bbc28c1742bfe88c9f0af8ce13a3750ad7a Mon Sep 17 00:00:00 2001
From: Jay Berkenbilt <qjb@debian.org>
Date: Sun, 4 Oct 2026 11:15:04 +0800
Subject: [PATCH 4/4] DEBIAN: Link stdlibs to libicudata so we get reasonably
 sane ELF headers on armhf

Thanks Adam Conrad <adconrad@ubuntu.com>.  (Closes: #653457)

Signed-off-by: Jay Berkenbilt <qjb@debian.org>

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 icu4c/source/config/mh-linux | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/icu4c/source/config/mh-linux b/icu4c/source/config/mh-linux
index 53d6780d68b..2d49365011b 100644
--- a/icu4c/source/config/mh-linux
+++ b/icu4c/source/config/mh-linux
@@ -23,7 +23,7 @@ LD_RPATH= -Wl,-zorigin,-rpath,'$$'ORIGIN
 LD_RPATH_PRE = -Wl,-rpath,
 
 ## These are the library specific LDFLAGS
-LDFLAGSICUDT=-nodefaultlibs -nostdlib
+LDFLAGSICUDT=
 
 ## Compiler switch to embed a library name
 # The initial tab in the next line is to prevent icu-config from reading it.
-- 
2.56.0

